src/main/java/de/ait/secondlife/config/DOProperties.java

```diff
@@ -0,0 +1,18 @@
+package de.ait.secondlife.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+@ConfigurationProperties(prefix = "do")
```

Comment:
I would put all the environment variables in the application.properties file and pull them from there. with this approach, deploying the application is much more convenient - enough to look into the properties and understand what exactly needs to be written into the OS.

Reply:
fix